test/integration/gossip_test.js

```diff
@@ -59,10 +59,11 @@ function mkBadPingReqResponder(ringpop) {
 }
 
 testRingpopCluster({
-    tap: function tap(cluster) {
-        mkNoGossip(cluster);
+    tapAfterConvergence: function tapAfterConvergence(cluster) {
```

Comment:
Why was this needed in the first place? We need gossip for the ring to converge. We could turn gossip off after convergence but I'm not sure why